views/partials/ui_node_script.hbs

```diff
@@ -6,4 +6,5 @@
   crossorigin="{{attributes.crossorigin}}"
   {{#if attributes.async}}async{{/if}}
   data-testid="node/script/{{attributes.id}}"
+  async="true"
```

Comment:
How does this work with `{{#if attributes.async}}async{{/if}}` a few lines above?